x/slashing/keeper/keeper_test.go

```diff
@@ -163,16 +163,15 @@ func TestHandleAlreadyJailed(t *testing.T) {
 	require.Equal(t, stakingtypes.Unbonding, validator.GetStatus())
 
 	// validator should have been slashed
-	resultingTokens := amt.Sub(app.StakingKeeper.TokensFromConsensusPower(ctx, 1))
-	require.Equal(t, resultingTokens, validator.GetTokens())
+	require.Equal(t, amt, validator.GetTokens())
```

Comment:
i dont think we should modify the tests we would like to still establish that the slashing works as expected if needed in the future, lets set the slashing percentage to the old defaults in the tests prior to test execution